symphony-app-authentication-fe/webpack.config.js

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable */
+var webpack = require('webpack');
+var path = require("path"),
+  CopyWebpackPlugin = require('copy-webpack-plugin');
+
+module.exports = {
+  entry: {
+    // babelPolyfill: 'babel-polyfill',
+    controller: path.resolve(__dirname, "./src/main/webapp/js/controller.js")
+  },
+  output: {
+    path: path.resolve(__dirname, "./target/static"),
+    filename: "[name].bundle.js"
+  },
+  devtool: 'source-map',
+  module: {
```

Comment:
Please review this config file. There are a lot of stuff not required like react presets, handlebars, css, images, fonts, etc.

Reply:
Done.